src/app/about/page.js

```diff
@@ -2,7 +2,7 @@ import AboutUs from "@/components/AboutUs/AboutUs";
 
 const AboutUss = () => {
   return (
-    <div className="h-screen">
+    <div>
       <AboutUs />
     </div>
```

Comment:
use arrow  and don't keep unnecessary code on codebase please do like this:
`const AboutUss = () => <AboutUs>
export default AboutUss
`